bread/layout/components/search_select.py

```diff
@@ -0,0 +1,109 @@
+import htmlgenerator as hg
+from django.utils.translation import gettext_lazy as _
+
+import bread.utils
+
+from .icon import Icon
+from .loading import Loading
+from .tag import Tag
+
+
+class SearchSelect(hg.BaseElement):
+    def __init__(
+        self,
+        search_view,
```

Comment:
I think this should be a URL and not a viewname. Using reverse inside a layout component is likely suboptimal because we can only use django-urls without parameters. But we might want to use an external URL or a django view with parameters in the path. These decisions should in general not be done in the components if we can avoid. The caller/user should make this decision.

Reply:
will take care of it. resolving this conversation as we have another one open regarding the bigger issue.